### DISCUSSION TITLE
ERROR: deployment failed: Quota Issue and invalid template

### DISCUSSION THREAD

**Discussion by IsaacGemal** [microsoft/AI-Chat-App-Hack#60] [Q&A] [answered]
Hi,

First time azure user here. 
I'm currently going through [this ](https://moaw.dev/workshop/?src=gh%3Aazure-samples%2Fazure-openai-rag-workshop%2Fbase%2Fdocs%2F&step=4#)guide -
I had just run `azd provision` and I got these two error messages.

> {
>   "code": "InvalidTemplateDeployment",
>   "message": "The template deployment 'openai' is not valid according to the validation procedure. The tracking id is 'faf97701-5dc3-4ee0-83bd-e68ac82c22db'. See inner errors for details.",
>   "details": [
>     {
>       "code": "InsufficientQuota",
>       "message": "This operation require 30 new capacity in quota Tokens Per Minute (thousands) - GPT-35-Turbo, which is bigger than the current available capacity 1. The current quota usage is 0 and the quota limit is 1 for quota Tokens Per Minute (thousands) - GPT-35-Turbo."
>     }
>   ]
> }

I'm wondering if the quota issue has something to do with the fact that I'm running off the $200 30 day free trial of azure?
Now talking to chatgpt my guess was that I have is that I have to go to subscriptions --> select subscription --> usage + quota,  but that's greyed out for some reason? Not sure exactly what's happening.

![chrome_eshcDvnrqQ](https://github.com/microsoft/AI-Chat-App-Hack/assets/147355120/e5f004c3-926b-462a-be5a-11de74596c99)

Thank you!

**Comment by pamelafox** [comment, marked as answer]
You can try changing capacity to 1 TPM in main.bicep (versus the current value of 30 - search for that) or use an OpenAI.com OpenAI account, per https://github.com/Azure-Samples/azure-search-openai-demo/?tab=readme-ov-file#openaicom-openai

**Reply by pamelafox** [reply]
You can also try opening up Azure OpenAI Studio and looking at the Quotas tab to see if you have other OpenAI deployments that are using up all your quota, or if you truly only have 1 TPM for your whole account.

**Reply by IsaacGemal** [reply]
So far, this seems to have worked! Thanks
